src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -72,10 +86,18 @@ public ObjectResult(object value)
             OutputFormatterContext formatterContext,
             IEnumerable<IOutputFormatter> formatters)
         {
+            var logger = formatterContext.ActionContext.HttpContext.RequestServices
+                .GetRequiredService<ILoggerFactory>()
+                .CreateLogger<ObjectResult>();
+
             // Check if any content-type was explicitly set (for example, via ProducesAttribute 
             // or Url path extension mapping). If yes, then ignore content-negotiation and use this content-type.
             if (ContentTypes.Count == 1)
             {
+                logger.LogVerbose(
+                    "Skipped content-negotiation as content type '{ContentType}' is explicitly set for the response.", 
```

Comment:
Should we bother with this? It's pretty much just telling you what code you wrote.